src/UserSpaceInstrumentation/AttachTest.cpp

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <sys/types.h>
+
+#include <set>
+#include <vector>
+
+#include "DummyProcessForTest.h"
+#include "OrbitBase/GetProcessIds.h"
+#include "UserSpaceInstrumentation/Attach.h"
+#include "absl/time/clock.h"
+#include "gtest/gtest.h"
+
+namespace orbit_user_space_instrumentation {
+
+TEST(AttachTest, AttachAndStopProcess) {
+  DummyProcessForTest dummy_process;
+  const pid_t pid = dummy_process.pid();
+
+  const auto result = AttachAndStopProcess(pid);
+  ASSERT_TRUE(result) << result.error().message();
+
+  // Verify that no new threads get spawned - i.e. the process is not running anymore.
```

Comment:
This check works because the dummy process is spawning threads all the time, right? This comment reads a bit like it's possible to check if a process is running by checking if it keeps spawning threads.

Reply:
adjusted comment